libensemble/resources/resources.py

```diff
@@ -22,40 +26,91 @@ class ResourcesException(Exception):
 class Resources:
     """Provides system resources to libEnsemble and executor.
 
-    This is intialized when the executor is created with auto_resources set to true.
+    A resources instance is always intialized unless ``libE_specs['disable_resource_manager']`` is True.
+
+    **Class Attributes:**
+
+    :cvar Resources: resources: The resources object is stored here and can be retrieved in user functions.
 
     **Object Attributes:**
 
     These are set on initialization.
 
+    :ivar string top_level_dir: Directory where searches for node_list file.
+    :ivar GlobalResources glob_resources: Maintains resources available to libEnsemble.
+
+    The following are set up after manager/worker fork.
+
+    The resource manager is set up only on the manaager, while the worker resources object is set
+    up on workers.
+
+    :ivar ResourceManager resource_manager: An object that manages resource set assignment to workers.
+    :ivar WorkerResources worker_resources: An object that contains worker specific resources.
+    """
+
+    resources = None
+
+    DEFAULT_NODEFILE = 'node_list'
+
+    @classmethod
+    def init_resources(cls, libE_specs):
+        """Initiate resource management"""
+
+        # If disable_resource_manager is True, then Resources.resources will remain None.
+        disable_resource_manager = libE_specs.get('disable_resource_manager', False)
+        if not disable_resource_manager:
+            top_level_dir = os.getcwd()  # SH TODO: Do we want as libE_specs option.
```

Comment:
I think I'm okay with this not being a libE_spec

Reply:
Yep